<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_SQL_Insert</class>
 <widget class="QDialog" name="Dialog_SQL_Insert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>311</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="text">
     <string>ОК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget_Students">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>461</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Insert_rand_all">
    <attribute name="title">
     <string>Страница</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_algoritms">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>(не выбрано)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Вихрь Мерсенна</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Макларен - Марсальи</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Использовать алгоритм</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>создания</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_kolvo_new_strok">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>строк в таблице</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>для</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>текущей базы данных.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Тип данных INT: пределы генерации от</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Тип данных VARCHAR: длина последовательности</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_int_left">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>110</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>до</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_int_right">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>110</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_varchar_lenght">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>140</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Тип данных REAL: пределы генерации от</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_real_left">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>170</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>до</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_real_right">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>170</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Тип данных BYTEA: пределы генерации от          до</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_bytea_left">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_bytea_right">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>200</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Тип данных BOOLEAN: использовать генерацию?</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_useBoolGen">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>230</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Да</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tableNow">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Студенты</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер варианта (0 - автоматическая генерация)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ФИО</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID студента (0 - автоматическая генерация)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>301</width>
       <height>81</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Группа</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>301</width>
       <height>81</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>230</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Задания</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID задания (0 - автоматическая генерация)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Текст</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="taskText">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="taskVariant">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>220</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="idTask">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Сложность (1 - легко, 2 - средне, 3 - сложно)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер варианта (0 - автоматическая генерация)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="taskComplication">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
